### PR TITLE
Remove build dependencies from final image (multistage build)

### PR DESCRIPTION
**Pull request by awigen**
This multistage build takes the resulting image size down from 346MB to
90MB.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -9,3 +8,8 @@ RUN apk update && apk add go git gcc musl-dev ca-certificates mailcap \
         && go build -x -ldflags "-X main.githash=$(git log --pretty=format:'%h' -n 1)" -o /bin/matterbridge \
         && rm -rf /go \
         && apk del --purge git go gcc musl-dev
+
+FROM alpine:edge
+RUN apk --no-cache add ca-certificates
```
Comment (42wim) [inline Dockerfile]:
```suggestion
RUN apk --no-cache add ca-certificates mailcap
```
We need mailcap for mimetypes: see  #969

**Comment by 42wim** [comment]
Thank you, good idea!

File: Dockerfile (hunk 1 above)
Comment (awigen) [reply Dockerfile]: I was unsure if that was a build dependency or not. I'll update the branch.

**Comment by 42wim** [comment]
Thanks!

**Pull request merged**